home/api/team.php

```diff
@@ -173,6 +180,17 @@ function cleanTeam()
     null,
     []
 );
+addMember(
+    1699875,
+    "Remyria",
+    "Alumni",
```

Comment:
Maybe worth adding his former position too? For example `Alumni (Community Manager)`